observations/obs_converters/run_tests.csh

```diff
@@ -193,12 +254,34 @@ foreach quickb ( `find . -name quickbuild.csh -print` )
          # if we miss any programs which need input and we don't have a .in file, have it
          # read from /dev/null so it errors out and doesn't just sit there waiting for input
          if ( -f ../work/${PROG}.in ) then
-           ( ./$PROG < ../work/${PROG}.in > ${LOGDIR}/runlog.${project}.${PROG}.out ) || set FAILURE = 1
+
+           if ( -f using_mpi_for_$PROG ) then
+              ( ${MPICMD} ./$PROG < ../work/${PROG}.in > ${LOGDIR}/runlog.${project}.${PROG}.out ) || set FAILURE = 1
+           else
+              (           ./$PROG < ../work/${PROG}.in > ${LOGDIR}/runlog.${project}.${PROG}.out ) || set FAILURE = 1
+           endif
```

Comment:
Here's the part where its testing for the presence of the file created by the mkmf_gsi_to_obs and then
invoking the mpicmd.  Identical philosophy to elsewhere, caveat this directory has some programs that also take command-line arguments.   In a testing framework, we just redirect STDIN from the ${PROG.in} file ... which is
in the dart test tarfile ...